src/mrpro/operators/LinearOperator.py

```diff
@@ -47,6 +48,71 @@ def H(self) -> LinearOperator:  # noqa: N802
         """Adjoint operator."""
         return AdjointLinearOperator(self)
 
+    def operator_norm(
+        self,
+        initial_value: torch.Tensor,
+        dim: Sequence[int] | None,
+        max_iterations: int = 64,
```

Comment:
should actually be n_max_iterations but it also different in cg and adams/lbfgs...